### PR TITLE
Gtransform 338 add external link tag and directory components update

### PR DESCRIPTION
**Pull request by jeanettemariemurphy**
To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.
 - [x] (if updating requirements) Requirements have been compiled.

### CODE REVIEW

File: article/templates/article/article_detail_content.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% load parse_date from article_tags %}
-{% load add_export_elements_classes breadcrumbs from directory_components %}
+{% load add_export_elements_classes add_href_target breadcrumbs from directory_components %}
```
Comment (MichaelWalker) [inline article/templates/article/article_detail_content.html]: Is the intention of this tag to cause external links to open in a new tab?

Is there any chance to change the name to make its intention a little more obvious (I might be a bit late to suggest that - sorry)

File: article/templates/article/country_guide.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {% extends 'article/base.html' %}
 {% load static %}
-{% load breadcrumbs card image_with_caption cta_link cta_card add_export_elements_classes convert_headings_to override_elements_css_class statistics_card_grid from directory_components %}
+{% load breadcrumbs card image_with_caption cta_link cta_card add_export_elements_classes convert_headings_to override_elements_css_class statistics_card_grid add_href_target from directory_components %}
```
Comment (MichaelWalker) [inline article/templates/article/country_guide.html]: We don't seem to use this import.

File: article/templates/article/article_detail_content.html (hunk 1 above)
Comment (jeanettemariemurphy) [reply article/templates/article/article_detail_content.html]: Yes, this tag was already existing, originally from directory-ui-supplier, but as far as I could see it wasn't used before I moved it to directory components, so could rename it.  I could look at this as an enhancement if ok?

Comment (MichaelWalker) [reply article/templates/article/article_detail_content.html]: yeah of course - not really anything to do with this PR, so shouldn't hold it up! Just saw it and didn't have any idea what it did from the name! 😊

**Pull request merged**